### ISSUE TITLE
[bug] sequence_type process fails for Acinetobacter

### ISSUE THREAD

**Issue by embatty** [bactopia/bactopia#207]
**Description**
MLST typing fails due to an extra column in the allelic profiles for Acinetobacter baumannii.

**Steps to Reproduce**
Run bactopia using the default Acinetobacter baumannii datasets, and it fails during the sequence_type task with the following error:
```Command error:
  Traceback (most recent call last):
    File "/miniconda3/envs/bactopia/share/bactopia-1.6.x/bin/mlst-blast.py", line 185, in <module>
      verbose=not args.quiet, compressed=args.compressed)
    File "/miniconda3/envs/bactopia/share/bactopia-1.6.x/bin/mlst-blast.py", line 64, in blast_alleles
      alleles.append(f'{name}.{cols[i]}')
  IndexError: list index out of range
```

I think the error is because the [Acinetobacter baumannii Oxford MLST profiles](https://pubmlst.org/bigsdb?db=pubmlst_abaumannii_seqdef&page=downloadProfiles&scheme_id=1) have a species column, which is empty for some profiles. If I change it to ignore the species column, it works fine.

- Bactopia Version: 1.6.1
- OS: CentOS
- Environment: conda on UGE cluster

**Comment by rpetit3** [comment]
Thanks for bringing this up! I'll get this fixed and updated soon

**Comment by rpetit3** [comment]
Thanks again @embatty for bringing this up. I'm sorry you ran into the error, but also kind of glad! 

I wasn't aware some profiles included a `species` column. I'm going to make some changes to first fix your error, but more importantly include the values for the `clonal_complex` and `species` columns in the results. At the moment they are tossed, and that seems pretty annoying to have to go back and manually look up the CC (and in your case the species.)

**Comment by rpetit3** [comment]
Ok submitted a fix: https://github.com/bactopia/bactopia/commit/1b1f0f5bd0f76521dbe135c87ce43097bbfa3d86

Also added extra fields to the output:
```
"ST": {
    "st": "931",
    "perfect_matches": 7,
    "clonal_complex": "",
    "species": ""
}
```

I'm going to test some more and likely push a version update in the next few days

**Comment by rpetit3** [comment]
Hi @embatty 

I released v1.6.3, which includes the changes to `mlst-blast`. I think this should take care of things for you.

Please let me know if not.

Thank you!
Robert

**Comment by rpetit3** [comment]
Going to close, feel free to reopen
